integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/testsupport/stub/MapperEasyMockUtils.java

```diff
@@ -91,4 +101,22 @@ private MapperEasyMockUtils() {
 			return this;
 		}
 	}
+
+	private static <R, E> void loadWithoutTimeout(EntityLoader<R, E> objectLoaderMock,
+			HitMappingDefinitionContext<R, E> context) {
+		expect( objectLoaderMock.loadBlocking(
+				EasyMockUtils.collectionAnyOrderMatcher( new ArrayList<>( context.loadingMap.keySet() ) ), isNull() ) )
+				.andAnswer( () -> ( (List<R>) EasyMock.getCurrentArguments()[0] ).stream()
+						.map( context.loadingMap::get )
+						.collect( Collectors.toList() ) );
+	}
+
+	private static <R, E> void loadWithTimeout(EntityLoader<R, E> objectLoaderMock,
+			HitMappingDefinitionContext<R, E> context) {
+		expect( objectLoaderMock.loadBlocking(
+				EasyMockUtils.collectionAnyOrderMatcher( new ArrayList<>( context.loadingMap.keySet() ) ), notNull() ) )
```

Comment:
Right, good idea. You could even make this more powerful and pass a `Capture<Long>`, so that in your tests, you can check that the number is within a given range.

Reply:
yep makes sense to use a spy for that